src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -397,6 +439,14 @@ by { ext, exact f.map_zero }
 @[simp] lemma zero_comp (f : normed_group_hom V₁ V₂) : (0 : normed_group_hom V₂ V₃).comp f = 0 :=
 by { ext, refl }
 
+lemma comp_assoc {V₄: Type* } [semi_normed_group V₄] (h : normed_group_hom V₃ V₄)
+  (g : normed_group_hom V₂ V₃) (f : normed_group_hom V₁ V₂) :
+  (h.comp g).comp f = h.comp (g.comp f) :=
+by { ext, refl }
+
+lemma coe_comp (f : normed_group_hom V₁ V₂) (g : normed_group_hom V₂ V₃) :
```

Comment:
Should this be a `simp`-lemma?
```suggestion
@[simp] lemma coe_comp (f : normed_group_hom V₁ V₂) (g : normed_group_hom V₂ V₃) :
```

Reply:
I went back to the non `simp` version since the linter was not happy: `normed_group_hom.comp_apply` (that is autogenerated) is not in `simp` normal form.